crates/lumina_shared/src/player/spawn_point.rs

```diff
@@ -10,43 +10,50 @@ impl Plugin for SpawnPointPlugin {
     fn build(&self, app: &mut App) {
         app.add_systems(
             PostUpdate,
-            init_spaceship.after(TransformSystem::TransformPropagate),
+            init_spaceships_at_spawn_points.after(TransformSystem::TransformPropagate),
         )
         .observe(on_add_spawned)
         .observe(on_remove_spawned);
     }
 }
 
-// Initialize spaceships and assign them alternately to teams.
-fn init_spaceship(
+/// Initialize spaceships and assign them to available spawn points.
+fn init_spaceships_at_spawn_points(
     mut commands: Commands,
-    q_spawn_points: Query<(&GlobalTransform, Entity), (With<SpawnPoint>, Without<SpawnPointUsed>)>,
-    mut q_spaceships: Query<
+    spawn_point_query: Query<
```

Comment:
```suggestion
    q_spawn_point: Query<
```

retain the project's convention.